src/card.css

```diff
@@ -0,0 +1,98 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  --color-grey-font: #616070;
+  --color-black-font: #060b35;
+  --color-blue: #00acdc;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__product-img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 40px 16px 0;
+  color: var(--color-black-font);
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  margin: 4px 16px 0;
```

Comment:
instead of setting horizontal margin for every element you can set padding for the whole card